srcpkgs/xonotic-data-low/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'xonotic-data-low'
 pkgname=xonotic-data-low
 version=0.8.2
-revision=2
+revision=3
```

Comment:
we don't need to revbump if the checksum is the same, there shouldn't be a change in the resulting package